packages/terra-form-checkbox/docs/UPGRADEGUIDE.md

```diff
@@ -1,5 +1,12 @@
 # Terra Form Checkbox Upgrade Guide
 
+## Changes from version 3 to version 4
+
+### Props
+
+#### Updated
```

Comment:
The Props and Updated sections might need to be reworked to make the description below make a little more sense. I think you had a section before that listed the changes. I think it helps to list the changes and then providing description/guidance for upgrading. 

"What happened?"

"Here's how to upgrade"

Reply:
Updated in b447243